en/ch03.md

```diff
@@ -1,17 +1,58 @@
+
+### Daemon synchronization with remote node
+If you use a public remote node or "simple mode", your daemon will use the blockchain of a third party remote note. Since it will not have to download the whole blockchain file, your synchronization will be faster.
+
+### Block height
+After your daemon and your wallet are synchronized, your daemon progress bar will display "Daemon is synchronized (number)", see below:
+
+![sync_remotenode](media/sync_remotenode.png)
+
+This number between parenthesis is the current block height, which is the latest block received by your daemon. In average, this number wil increase +1 every 2 minutes, when a new block is found by a miner.
+
+## Wallet synchronization
+After synchronizing the daemon, your wallet will scan the blockchain file with its private view key, searching for past transactions. Once a previous transaction is detected and retrieved by your wallet, your balance and your transaction history will be updated.
+
+Since all transactions are encrypted in Monero, your wallet will have to decrypt a lot of transactions. This process places very high demands on your computer's resources, including CPU processing and many readings and writings in your hard disk. Depending on your system configuration, this can take a long time, even days, so please be patient. Be aware that the synchronization speed is not always constant. Some blocks can have more transactions and/or use different technologies, therefore they may take longer to verify.
+
+### Wallet restore height
+The wallet restore height is the block number (block height) where your wallet will start searching for transactions.
+This number should be a block height before the block that included your first incoming transaction.
+For example, if your wallet received Monero for the first time in 2020-09-20, your wallet restore height must be a block number before the block height of this date. Monero wallets usually ask you to write down your wallet restore height together with your restore seed when you are creating a wallet.
+
+### How to change Wallet restore height / Wrong or zero balance after synchronizing
+In order to see the correct balance, your wallet must be fully synchronized AND using a correct restore height.
+If your wallet is fully synchronized but your balance is still wrong, you're probably using a wallet restore height that is higher than the block height of your first incoming transaction.
+To solve this, you have to set a new wallet restore height. Go to Settings > Info page, Wallet restore height and click on "(Change)".
+
+![change_wallet_restore_height](media/change_wallet_restore_height.png)
+
+Next, enter a new restore height or an estimate date before your first incoming transaction, and then click on "Ok".
+
+![set_new_restore_height](media/set_new_restore_height.png)
+
+Your wallet will now rescan the blockchain starting from the new restore height.
```

Comment:
This section is better suited for the `common issues and solutions` chapter.

Reply:
I think we should keep this in this section, and add a link to this chapter in the common issues and solutions chapter.

Reply:
Yeah, that would be fine. Could you add the link to the "common issues" chapter? Otherwise i'll do it in another PR.

Reply:
I don't know how to add a link to a page inside the document. I think you'll have to add it later when creating the .pdf file.

Reply:
No problem. Added with #188